sources/index.yml

```diff
@@ -10,12 +10,12 @@ orbs:
 executors:
   node:
     docker:
-      - image: circleci/node:12.22-buster
+      - image: cimg/node:12.22
```

Comment:
Should we update this? I'm worried it'll have knock-on-effects if any projects don't specify a node version and require an old version, but the only old projects I can think of are old backend services, which are typically easy to update.

Open to ideas on which version to default to (latest LTS)?

Reply:
I'd vote for getting this to the most recent version we can without too much pain.

Reply:
Agree. If old services are stuck on 12 for some reason then we should add a `node12` executor for them and then figure out how to get them off of it.